07-Customizing-AzSK-for-your-Org/Readme.md

```diff
@@ -460,7 +460,36 @@ image below:
 This change will be effective across your organization immediately. Anyone running AzSK commands (in fresh PS sessions)
 should see the new message. 
 
-##### b) Changing a control setting for specific controls 
+##### b) Customize output folder for my org
+Normally, when we run GSS the security scan results go to a default folder 
+`%LOCALAPPDATA%\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]E.g.`
+C:\Users\UserName\AppData\Local\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]\20170331_142819. 
+This location can be customized in an org policy so that the scan results can be collected at a particular location on org policy users machine.
+
+###### Steps:
+
+i) Open the AzSk.json from your local org-policy folder. Path for example: D:\ContosoPolicies\Config\AzSK.json
```

Comment:
'D:\ContosoPolicies\Config\AzSK.json' in this instead of D you can put like <path where the downloaded policy files are present>